tests/integration/test_relation.py

```diff
@@ -43,7 +43,7 @@ async def build_and_deploy(ops_test: OpsTest, run_action):
 
     async def build_and_deploy_ingress():
         charm = await ops_test.build_charm(".")
-        return await ops_test.model.deploy(
+        return await ops_test.model.deploy(  # type: ignore[union-attr]
```

Comment:
How about adding an `assert ops_test.model` somewhere earlier so that `ops_test.model` is guaranteed not to be `None` in the test?

Reply:
My first attempt didn't work out but now it's ok, thanks.